lib/dslash_quda.cu

```diff
@@ -98,6 +125,27 @@ namespace quda {
       cudaEventCreateWithFlags(&packEnd[i], cudaEventDisableTiming);
       cudaEventCreateWithFlags(&dslashStart[i], cudaEventDisableTiming);
     }
+#ifdef NVSHMEM_COMMS
+    sync_arr = (shmem_sync_t *)device_comms_pinned_malloc(2 * QUDA_MAX_DIM * sizeof(shmem_sync_t));
+    cudaMemset(sync_arr, 0, 2 * QUDA_MAX_DIM * sizeof(shmem_sync_t));
```

Comment:
`cudaMemset` -> `qudaMemset`? (I feel like there was some fuzziness about `qudaMemset` here or there, thus the "?")